packages/browser-repl/src/components/types/error-output.tsx

```diff
@@ -31,7 +31,7 @@ export class ErrorOutput extends Component<ErrorOutputProps> {
     if (isShouldReportAsBugError(this.props.value)) {
       return (<div>
         This is an error inside mongosh.
-        Please <a href="https://jira.mongodb.org/projects/MONGOSH/issues" target="_blank">file a bug report for the MONGOSH project</a>.
+        Please <a href="https://jira.mongodb.org/projects/MONGOSH/issues" target="_blank" rel="noreferrer">file a bug report for the MONGOSH project</a>.
```

Comment:
I just noticed --fix added rel="noreferrer" to links. Do we actually want this?

Reply:
I think it's okay to keep although not that it's required in our case, I guess it never got triggered in Compass because we always use Link component instead? 🤔